### ISSUE TITLE
Singleboostr wont find the game i want to boost

### ISSUE THREAD

**Issue by ntrr** [Ezzpify/HourBoostr#119]
Is there a way to manually add a game so it can be afk'd? 

Thanks in advance and thanks for the awesome program!

**Comment by Ezzpify** [comment]
Go to settings and download a new game list

https://i.imgur.com/YRdJrRA.png

**Comment by ntrr** [comment]
Yeah ive done that. The game i specifically wanted to use was a test server game. I just figured all test server games would work since 1 of them did.

**Comment by Ezzpify** [comment]
You can add the ID to the app list manually and try if that works

**Comment by ntrr** [comment]
How does one do that? I was looking to try to do that, but couldnt find out how
